theano/sandbox/cuda/dnn_base.c

```diff
@@ -5,31 +5,44 @@ static cudnnHandle_t _handle = NULL;
 static int
 c_set_tensorNd(CudaNdarray *var, cudnnTensorDescriptor_t desc) {
 
-
   int dim = CudaNdarray_NDIM(var);
-  int strides[dim];
+  int *strides = (int *)malloc(dim * sizeof(int));
```

Comment:
Can we use calloc on windows? I think it is faster on linux.

Otherwise, thanks.